README.md

```diff
@@ -175,7 +175,4 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-A copy of the license is available in the repository's [LICENSE](./license.txt) file.
-
-[](Esri Tags: ArcGIS Web Mapping Leaflet AppBuilder Hackerlabs Labs JavaScript)
-[](Esri Language: JavaScript)
```

Comment:
that small little change in github broke a ton of markdown pages on github in general.